src/Uno.UI/UI/Xaml/Controls/WebView/NativeWebView.iOS.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Windows.UI.Xaml.Controls
+{
+	public partial class NativeWebView : UnoWKWebView
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['partial' is gratuitous in this context.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826539)